File: contract/src/main/java/rrlane/leasing/contract/dto/VehicleDTO.java
package rrlane.leasing.contract.dto;

import lombok.Data;
import lombok.RequiredArgsConstructor;

@RequiredArgsConstructor
@Data
public class VehicleDTO {
    private String brand; //prop: company brand
    private String model; //prop: model name
    private String myear; //prop: model year
    private String vin; //prop: vehicle identification number
    private Double price;

    @Override
    public String toString() {
        return "VehicleDTO{" + "brand='" + brand + '\'' + ", model='" + model + '\'' + ", year='" + myear + '\'' + ", vin='" + vin + '\'' + ", price=" + price + '}';
    }
}

File: clientng/src/app/vehicle/dto/VehicleDTO.ts
export class VehicleDTO {
  
  constructor(private _brand: string,private _model: string,private _makeYear: string,private _vin: string,private _price: number) {}

  public get brand(): string {
    return this._brand
  }

  public get model(): string {
    return this._model
  }
  
  public get makeYear(): string {
    return this._makeYear
  }
  
  public get vin(): string {
    return this._vin
  }
  
  public get price(): number {
    return this._price
  }
}